tests/Tags/ChildrenTest.php

```diff
@@ -80,4 +82,19 @@ public function it_gets_children_data_when_in_another_site()
 
         $this->assertEquals('the french bar entry', $this->tag('{{ children }}{{ title }}{{ /children }}'));
     }
+
+    /** @test */
+    public function it_doesnt_affect_children_in_nav()
+    {
+        $this->setUpEntries();
+
+        $mock = Mockery::mock(Children::class);
+        $mock->shouldNotReceive('index');
```

Comment:
This mock will never have any methods called on it because it wouldn't be the same one that gets resolved out of the container.

You can probably do this:

```php
$this->app['statamic.tags']['children'] = $mock;
```

Reply:
Yep makes sense, thanks